types/rezervo.ts

```diff
@@ -93,8 +91,39 @@ export type RezervoDaySchedule = {
     classes: RezervoClass[];
 };
 
-// This will eventually be implemented as its own type, but for now it ports to SitClass
-export type RezervoClass = SitClass;
+export type RezervoClass = {
+    id: number;
+    activityId: number;
+    available: number;
+    bookable: boolean;
+    capacity: number;
+    studio: {
+        id: number;
+        name: string;
+    };
+    room: string;
+    from: string;
+    to: string;
+    name: string;
+    description: string;
+    category: {
+        id: string;
+        name: string;
+        excelineId: number;
```

Comment:
I think `excelineId` is only relevant for Sit, and since we don't use it right now it should be removed.

Reply:
Should be removed from the Sit type as well :hugs: